rafthttp/http.go

```diff
@@ -125,6 +125,10 @@ func (h *streamHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	w.Header().Add("X-Rafthttp-Stream", string(streamTypeMsgApp))
```

Comment:
done in https://github.com/yichengq/etcd/commit/ea7251c3476c498ecac8d7531727a2dd07ab25ae